src/test/java/alinaTinca/tests/Tema5/Demoqa/Elements/WebTablesClass.java

```diff
@@ -0,0 +1,137 @@
+
+    public static void goToTheWebTables(String firstName, String lastName, String email, String age, String salary, String department){
+        //create a method called goToTheWebTables that identifies several elements from Web Tables section and makes operations on these elements
+        ChromeDriver driver = BrowserManagerClass.createChromeDriver();
+        //create and configure the Chrome browser page settings
+        driver.get("https://demoqa.com/webtables");
+        //go to the Demoqa URL
+        WebElement WebTablesSection = driver.findElement(By.cssSelector("#item-3"));
+        //find the section called Web Tables
+        WebTablesSection.click();
+        //click on the section called Web Tables
+        WebElement AddButton = driver.findElement(By.cssSelector("#addNewRecordButton"));
+        //find the Add Button
+        AddButton.click();
+        //click on the Add Button
+        WebElement firstNameField = driver.findElement(By.cssSelector("input[placeholder='First Name']"));
+        //find the First Name field
+        firstNameField.click();
+        //click on the First Name field
+        firstNameField.sendKeys(firstName);
+        //introduce new first name
+        WebElement lastNameField= driver.findElement(By.cssSelector("input[placeholder='Last Name']"));
+        //find the Last Name field
+        lastNameField.click();
+        //click on the Last Name field
+        lastNameField.sendKeys(lastName);
+        //introduce new last name
+        WebElement emailField = driver.findElement(By.cssSelector("input[placeholder='name@example.com']"));
+        //find the Email Field
+        emailField.click();
+        //click on the Email field
+        emailField.sendKeys(email);
+        //introduce new email
+        WebElement ageField = driver.findElement(By.cssSelector("input[placeholder='Age']"));
+        //find the Age field
+        ageField.click();
+        //click on the Age field
+        ageField.sendKeys(age);
+        //introduce new age
+        WebElement salaryField = driver.findElement(By.cssSelector("input[placeholder='Salary']"));
+        //find the Salary field
+        salaryField.click();
+        //click on the Salary field
+        salaryField.sendKeys(salary);
+        //introduce new salary
+        WebElement departmentField = driver.findElement(By.cssSelector("input[id='department']"));
+        //find the Department field
+        departmentField.click();
+        //click on this field
+        departmentField.sendKeys(department);
+        //introduce new department
+        WebElement SubmitButton = driver.findElement(By.cssSelector("button[id='submit']"));
+        //find the button called Submit
+        SubmitButton.click();
+        //click on Submit button
+        WebElement deleteIcon = driver.findElement(By.cssSelector(" span[id='delete-record-4']"));
+        //find the Bin element gaseste iconul Bin (si sterge randul 4 adaugat anterior)
+        deleteIcon.click();
```

Comment:
You'll most probably have to scroll down to be able to delete. There is an add that covers that row sometimes. 
![image](https://user-images.githubusercontent.com/18215854/211500709-6a4cf131-f7ef-41c7-89ab-95f6dc4e95e9.png)